progress/upload_status.py

```diff
@@ -38,6 +39,16 @@ def get_directory_status(directory, required_file_list):
         return result
 
     file_list = next(os.walk(directory))[2]  # Gets the list of files in the directory
+
+    # Legacy upload catch
+    # When the irida-miseq-uploader (old uploader) ran it generated a .miseqUploaderComplete file
+    # To prevent uploading runs that used this old system, we assume runs with this file are COMPLETE
+    # They can still be uploaded using the --force option, but will not be picked up automatically with --batch
```

Comment:
Reword this to be more specific

Reply:
Fixed in effa23ba966ebf6801960a9a3075812ff0446a91